features/visibilityOfDialoguesTabsForTheAdmin.feature

```diff
@@ -0,0 +1,20 @@
+@visibilityOfDialoguesTabsForTheAdmin
+
+
+Feature: logging in as an administrator
+I want to be able to perform administration options (e.g. options related to the school)
+Background:
+	Given The admin arrives on the Schul-Cloud page
+	Given The admin is logged in successfully
+
+Scenario:
+    Then The admin is supposed to accept the data protection agreement
+    When The admin goes to the Administration page
+    Then Verify if all required tabs are visible in Administration area
+                | tabs   |
+                |SCHÜLER | 
```

Comment:
i've removed spaces (on the left side), this syntax allow that test runs and check all tabs during only one admin login (we save time for executing tests, that was suggestion for Yulia S.), otherwise test will be run six times and check in one test one tab.

Reply:
Cool, ist this documented?

Reply:
i'll ask Yulia and if it's needed, we'll create one.